src/include/duckdb/main/connection_manager.hpp

```diff
@@ -13,19 +13,28 @@
 
 #include <mutex>
 
+#ifdef _WIN32
+#define DUCKDBAPI __declspec(dllexport)
+#define DUCKDBCALL __cdecl
+#else
+#define DUCKDBAPI
+#define DUCKDBCALL
+#endif
+
 namespace duckdb {
 
 class DuckDB;
 class Connection;
 
 class ConnectionManager {
 public:
-	~ConnectionManager();
+	DUCKDBAPI ~ConnectionManager();
```

Comment:
The ConnectionManager is supposed to be internally used only; does this need to be exported?